onnxruntime/core/session/inference_session.cc

```diff
@@ -418,6 +420,40 @@ common::Status InferenceSession::RegisterCustomRegistry(std::shared_ptr<CustomRe
   return Status::OK();
 }
 
+common::Status InferenceSession::SaveToOrtFormat(const std::basic_string<ORTCHAR_T>& filepath) const {
+  ORT_RETURN_IF_NOT(FLATBUFFERS_LITTLEENDIAN, "ort format only supports little-edian machines");
+
+  // TODO, figure out a optimal buffer size than default 1024
+  flatbuffers::FlatBufferBuilder builder(1024);
+  auto ort_version = builder.CreateString(ORT_VERSION);
```

Comment:
1024 seems really small given this process will happen with a full build. I'd suggest 10MB or more given the size of models we're considering for this are often larger than that. that said, I'm not sure if there's a benefit to using a smaller size.

Reply:
Do we have a way to estimate a model size? 10MB seems a bit large for me

---
In reply to: [479745587](https://github.com/microsoft/onnxruntime/pull/4973#discussion_r479745587) [](ancestors = 479745587)